src/NSwagStudio/ViewModels/CodeGenerators/SwaggerToTypeScriptClientGeneratorViewModel.cs

```diff
@@ -73,9 +73,8 @@ public OpenApiToTypeScriptClientCommand Command
             .ToArray();
 
         /// <summary>Gets the list of date time types.</summary>
-        public TypeScriptDateTimeType[] DateTimeTypes { get; } = Enum.GetNames(typeof(TypeScriptDateTimeType))
-            .Select(t => (TypeScriptDateTimeType)Enum.Parse(typeof(TypeScriptDateTimeType), t))
-            .ToArray();
+        public TypeScriptDateTimeType[] DateTimeTypes { get; } =
+            (TypeScriptDateTimeType[])Enum.GetValues(typeof(TypeScriptDateTimeType));
```

Comment:
One line

Reply:
How about:
```suggestion
        public TypeScriptDateTimeType[] DateTimeTypes { get; } = Enum.GetValues<TypeScriptDateTimeType>();
```

Reply:
It makes very little difference to comment a merged PR, the work has been finished.

Reply:
According to the [documentation](https://learn.microsoft.com/en-us/dotnet/api/system.enum.getvalues?#system-enum-getvalues-1) the overloaded version of the generic method `GetValues` is available from the version .NET Core 5.0. 

The current [framework version of the project](https://github.com/RicoSuter/NSwag/blob/master/src/NSwagStudio/NSwagStudio.csproj#L4) `NSwagStudio` is .NET 4.6.2.

Reply:
> How about using the generic version [Enum.GetValues()](https://learn.microsoft.com/en-us/dotnet/api/system.enum.getvalues?view=net-8.0#system-enum-getvalues-1) ?

According to the [documentation](https://learn.microsoft.com/en-us/dotnet/api/system.enum.getvalues?#system-enum-getvalues-1) the overloaded version of the generic method GetValues is available from the version .NET Core 5.0.

The current [framework version of the project](https://github.com/RicoSuter/NSwag/blob/master/src/NSwagStudio/NSwagStudio.csproj#L4) NSwagStudio is .NET 4.6.2.

Reply:
Great catch, @testfirstcoder! 😅